src/test/kotlin/no/nav/faktureringskomponenten/service/avregning/AvregningBehandlerTest.kt

```diff
@@ -21,11 +23,15 @@ class AvregningBehandlerTest {
 
         val avregningsfaktura = avregningBehandler.lagAvregningsfaktura(fakturaseriePerioder, bestilteFakturaer)
 
-        avregningsfaktura.shouldNotBeNull()
-        avregningsfaktura.fakturaLinje shouldBe listOf(
+        val fakturalinjer = avregningsfaktura.run {
+            shouldNotBeNull()
+            shouldHaveSize(2)
+            flatMap { it.fakturaLinje }
```

Comment:
Å blande assert som `shouldHaveSize` på avregningsfakturaer med `flatMap` for linjer i samme block gjør koden vanskeligere å lese, synes jeg.
Jeg ville dele dette i 2